vault-rcp/src/main/java/org/apache/jackrabbit/vault/rcp/RcpTaskManager.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.jackrabbit.vault.rcp.impl;
+package org.apache.jackrabbit.vault.rcp;
```

Comment:
why has this changed?

Reply:
I will take the changes from rcp out -> tracked now in https://issues.apache.org/jira/browse/JCRVLT-438